youtube_dl/utils.py

```diff
@@ -248,6 +248,14 @@ def get_element_by_attribute(attribute, value, html):
     return unescapeHTML(res)
 
 
+def extract_attributes(attributes_str, attributes_regex=r'(?s)\s*([^\s=]+)\s*=\s*["\']([^"\']+)["\']'):
+    attributes = re.findall(attributes_regex, attributes_str)
+    attributes_dict = {}
+    if attributes:
+        attributes_dict = {attribute_name: attribute_value for (attribute_name, attribute_value) in attributes}
```

Comment:
the code is the same in #6811.
i just need an answer for https://github.com/rg3/youtube-dl/pull/6811#issuecomment-139608906.
and then i will fix it and keep it in single place.